### PR TITLE
DOCSP-35360: fix single index code snippet

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-35360>
Staging - https://preview-mongodbcchomongodb.gatsbyjs.io/golang/DOCSP-35360-fix-single-index-code/fundamentals/indexes/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/indexes.txt [hunk 1]
```diff
@@ -145,21 +145,21 @@ The following example creates an index in ascending order on the
    .. input::
       :language: go
 
+      coll := client.Database("sample_mflix").Collection("movies")
       indexModel := mongo.IndexModel{
-          Keys: bson.D{{"title", 1}}
+        Keys: bson.D{{"title", 1}},
```
Comment (ccho-mongodb) [inline source/fundamentals/indexes.txt:150]: This line contains the syntax fix.

File: source/fundamentals/indexes.txt [hunk 2]
```diff
@@ -91,14 +96,14 @@ Operational Considerations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To improve your query performance, create indexes on fields that appear
-often in your queries and operations that return sorted results. You
-should track index memory and disk usage for capacity planning since
+often in your queries and operations that return sorted results. Track
+index memory and disk usage for capacity planning since
 each index that you add consumes disk space and memory. In addition,
 when  a write operation updates an indexed field, MongoDB
```
Comment (rustagir) [inline source/fundamentals/indexes.txt:102]: S: noticed a double space here, fix

File: source/fundamentals/indexes.txt [hunk 3]
```diff
@@ -283,7 +290,7 @@ within the compound index.
 
    Text indexes differ from the more powerful
    :atlas:`Atlas full text search indexes </atlas-search>`.
-   Atlas users should use Atlas search.
+   We recommend that Atlas search for Atlas users.
```
Comment (rustagir) [inline source/fundamentals/indexes.txt:293]:
```suggestion
   We recommend Atlas search for Atlas users.
```

**Review by rustagir — APPROVED** [review]
lgtm but noticed some things

**Pull request merged**